components/AnalyticsProvider.tsx

```diff
@@ -0,0 +1,50 @@
+// import useDataDog from 'hooks/useAnalytics'
```

Comment:
remove this line, unless we plan on adding this hook?

Reply:
nice catch